pd/src/info.rs

```diff
@@ -66,31 +65,32 @@ impl Info {
         tracing::info!(?query);
 
         match query.path.as_str() {
-            "/jmt/key" => {
+            "state/key" => {
                 let height: u64 = query.height.into();
                 let key = query.data.to_vec();
 
-                let proof = jmt::JellyfishMerkleTree::new(&self.storage)
-                    .get_with_ics23_proof(key, height)
+                let jmt_proof = jmt::JellyfishMerkleTree::new(&self.storage)
+                    .get_with_ics23_proof(key.clone(), height)
                     .await?;
-                let value = proof.value.clone();
+                let value = jmt_proof.value.clone();
 
                 let commitment_proof = ics23::CommitmentProof {
-                    proof: Some(ics23::commitment_proof::Proof::Exist(proof)),
+                    proof: Some(ics23::commitment_proof::Proof::Exist(jmt_proof)),
                 };
 
-                let kvr = KeyValueResponse {
-                    value,
-                    proof: Some(commitment_proof),
+                let op = tendermint::merkle::proof::ProofOp {
+                    field_type: "jmt:v".to_string(),
+                    key,
+                    data: commitment_proof.encode_to_vec(),
```

Comment:
I'm surprised this is all we have to do -- where does `jmt:v` come from?

Reply:
https://github.com/cosmos/ibc-go/blob/main/modules/core/23-commitment/types/utils.go#L9 this is how relayers convert ABCI Proof to ICS23 proof